frontend/src/app/commonComponents/DatePicker.tsx

```diff
@@ -1,5 +1,9 @@
 import classnames from "classnames";
 import { DatePicker as TrussworksDatePicker } from "@trussworks/react-uswds";
+import React from "react";
```

Comment:
This was done because intellij recommended it - "Referenced UMD global variable"